src/model/ModelObject.cpp

```diff
@@ -376,6 +376,50 @@ namespace model {
       return result;
     }
 
+    /** Gets the fenestration value from the sql file **/
+    boost::optional<double> ModelObject_Impl::getFenestrationValue(std::string columnName) const {
+      boost::optional<double> result;
```

Comment:
Unless there is another class that will need this, this shouldn't be on ModelObject, but just on the SubSurface

Reply:
I find this method a bit confusing. Do we need this in addition to the better named `assemblyUFactor` etc?

Reply:
yeah but in that case it makes sense because lots of child classes get to use that method. Here that doesn't seem to be the case

Reply:
I don't see the harm - it reduces duplicated code. Should we rename the method to something other than `getFenestrationValue`?

Reply:
I changed it to `getExteriorFenestrationValue` so that it aligns with `TableName=Exterior Fenestration`.